src/components/Footer.tsx

```diff
@@ -1,80 +1,106 @@
-                    </p>
-                </div>
-                <div className="md:w-2/3 flex sm:flex-row flex-col gap-5 justify-center">
-                    <div>
-                        <h3 className="text-blue underline underline-offset-8 mb-3">Informations générales</h3>
-                        <ul className="flex flex-col gap-2">
-                            <li>
-                                <Link to="/about">Qui sommes-nous ?</Link>
-                            </li>
-                            <li>
-                                <Link to="/cgu">Conditions générales d'utilisation</Link>
-                            </li>
-                            <li>
-                                <Link to="/cgv">Conditions générales de vente</Link>
-                            </li>
-                            <li>
-                                <Link to="/gdpr">RGPD et gestion des cookies</Link>
-                            </li>
-                        </ul>
-                    </div>
-                    <div>
-                        <h3 className="text-blue underline underline-offset-8 mb-3">Des questions ?</h3>
-                        <ul className="flex flex-col gap-2">
-                            <li>
-                                <Link to="/help">Aides</Link>
-                            </li>
-                            <li>
-                                <Link to="/stripe">Le service de paiement</Link>
-                            </li>
-                            <li>
-                                <Link to="/security">Connexion et sécurité</Link>
-                            </li>
-                            <li>
-                                <Link to="/reservation">Le service de réservation</Link>
-                            </li>
-                        </ul>
-                    </div>
-                </div>
+    <footer className="py-8 text-gray-600 bg-gray-100">
+      <div className="container flex flex-col items-center px-4 mx-auto">
+        <div className="grid w-full grid-cols-1 gap-8 text-center md:grid-cols-2 lg:grid-cols-4 md:text-left">
+          {/* Logo and Company Info */}
+          <div className="flex flex-col items-center md:items-start">
+            <img src={Logo} alt="Share'loc Logo" width={250} height={100} />
+            <p className="mt-4 text-sm">
+              Postez vos annonces, louez des biens, partagez votre expérience,
+              générer des revenus grâce à vos biens innutilisés.
+            </p>
+          </div>
+
+          {/* General Information */}
+          <div className="flex flex-col items-center md:items-start">
+            <h3 className="mb-4 text-lg font-semibold">
+              Informations générales
+            </h3>
+            <ul className="space-y-2">
+              <li>
+                <Link to="/about" className="hover:text-gray-900">
+                  Qui sommes-nous ?
+                </Link>
```

Comment:
Doc about the new page keeps scroll : https://stackoverflow.com/questions/58431946/why-does-my-react-router-link-bring-me-to-the-middle-of-a-page